core/src/main/java/gov/nasa/jpl/mudrod/metadata/pre/ApiHarvester.java

```diff
@@ -53,6 +53,7 @@ public ApiHarvester(Properties props, ESDriver es, SparkDriver spark) {
   public Object execute() {
     LOG.info("Starting Metadata harvesting.");
     startTime = System.currentTimeMillis();
+    es.deleteType(props.getProperty("indexName"), props.getProperty("raw_metadataType"));
```

Comment:
Do you also need to remove all metadata-related files and ES types (e.g. metadatalinkagetype)? @quintinali